deployment/cdk/opensearch-service-migration/lib/network-stack.ts

```diff
@@ -288,9 +261,8 @@ export class NetworkStack extends Stack {
             });
 
             if (props.targetClusterEndpoint) {
-                const formattedClusterEndpoint = NetworkStack.validateAndReturnFormattedHttpURL(props.targetClusterEndpoint);
                 const deployId = props.addOnMigrationDeployId ? props.addOnMigrationDeployId : props.defaultDeployId;
-                createMigrationStringParameter(this, formattedClusterEndpoint, {
+                createMigrationStringParameter(this, props.targetClusterEndpoint, {
```

Comment:
Why did this line change?  From the rest of the PR, I can't figure that out.

Reply:
props.targetClusterEndpoint is already formatted from calling validateAndReturnFormattedHttpURL from parseClusterDefinition in stack-composer.

This now parsed/formatted endpoint that is in `targetCluster.endpoint` is passed as props.targetClusterEndpoint rendering formatting in this stack unnecessary.

Implementation LGTM